pages/index.ejs

```diff
@@ -14,32 +14,33 @@ description: Privacy Manager website
     </ul>
     <a id="install-button" href="https://chrome.google.com/webstore/detail/privacy-manager/giccehglhacakcfemddmfhdkahamfcmd" target=_blank>Get Privacy Manager for Chrome</a>
   </div>
-  <iframe class="tablet-hide end" id="promo-video" src="https://www.youtube.com/embed/iaKweivaOhI" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+  <iframe class="tablet-hide end" id="promo-media" src="https://www.youtube.com/embed/iaKweivaOhI" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
+  </iframe>
 </section>
 <section class="bg-secondary">
   <div class="col3 center container cards">
-    <h3 class="cell1-3">{what-you-get[Privacy Manager section h3 title] What <em>You Get</em>}</h3>
+    <h3 class="cell1-3">{what-you-get["What you get" section title] What <em>You Get</em>}</h3>
```

Comment:
Detail: Above for the similar string you are using `Header of the section `, thinking mentioning that it's `Header of the section` or title of the section is enough.

Reply:
Same applies to the sections headers below.

Reply:
Mean: Instead of `"What you get" section title ` `Section title` or `Header of the section `. And apply this logic accordingly to all section titles/headers.